Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -1936,24 +1938,18 @@ def get_remote_data_command(client: Client, args: Dict[str, Any], params: Dict)
 
     # get latest comments and files
     entries = []
-    attachments_res = client.get_ticket_attachments(ticket_id)
-    file_entries = client.get_ticket_attachment_entries(ticket.get('sys_id', ''))
-    if 'result' in attachments_res:
-        attachments = attachments_res['result']
-        for attachment in attachments:
+    file_entries = client.get_ticket_attachment_entries(ticket_id)
+    if file_entries:
+        for file_ in file_entries:
             entry_time = arg_to_timestamp(
-                arg=attachment.get('sys_created_on'),
+                arg=file_.get('sys_created_on'),
                 arg_name='sys_created_on',
                 required=False
             )
-
-            if file_entries:
-                for file_ in file_entries:
-                    if file_.get('File') == attachment.get('file_name'):
-                        if last_update > entry_time:
-                            continue
-                        else:
-                            entries.append(file_)
+            if last_update > entry_time:
```

Comment:
why is this condition needed?
we already check `if last_update > ticket_last_update`, does attachment not count as ticket update?

Reply:
No, its an entry. so we want to make sure that the attachment won't be fetched twice. 
We also have this check for comments entries

Reply:
checking it to be sure

Reply:
@Itay4 
So no, the attachments as the comments are entries and do not count as part of the ticket.
If I'll remove the condition, it will keep fetching all files over and over again.